spec/requests/api/v1/mentors_patch_request_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe 'mentors API' do
+  describe 'PATCH /api/v1/mentors/:id' do
+    it 'patches successfully to /api/v1/mentors/:id' do
+
+      id = create(:mentor).id
```

Comment:
great testing, as you mentioned, we can add a sad path, let's do this when we finish implementing JWT